### ISSUE TITLE
Integrate crypto benchmarks with CI

### ISSUE THREAD

**Issue by amc-ie** [MinaProtocol/mina#13518]

**Comment by fabrizio-m** [comment]
I think most of the benchmarks allow to output a JSON with the results, could that help for Grafana?

**Comment by shimkiv** [comment]
Yes I think Grafana can read from JSON (simple datasource). But we will anyway have to store it somewhere, hence, more generic approach will be better.
Publishing data into InfluxDB, for example, is very simple and I believe the JSON can be easily transformed into corresponding InfluxDB payload.

**Comment by dkijania** [comment]
After discussion with @fabrizio-m and @mrmr1993 the best approach would be to split effort into two items:

- Benchmarks in CI which will track regression . We don't necessary require super isolated VM as we are running baseline and then actual run.
- Nightly runs for proof systems benchmarks on isolated VM 
- Nightly runs for mina CI (end to end pickles run) on isolated VM
